src/styles/elements/buttons.scss

```diff
@@ -235,9 +235,8 @@
     }
 
     &--special {
-        color: #fff;
-        background: linear-gradient(-58deg, #49505c 0%, #3f51b5 36%, #00c7d6 100%);
-
+        color: #fff;       
+	background-color: #3f51bf;
```

Comment:
Could you use a color pre-defined in argocd styles instead of a color code

Reply:
> Also please remove the package.json and yarn.lock files

If I remove these files, again it will be visible in commit history?

Reply:
You'll want to use `git rebase -i origin/master` or similar, select the commits to `edit`, undo the changes, and then use `git commit --amend`